src/Sequence.cpp

```diff
@@ -962,13 +962,16 @@ void Sequence::WriteXML(XMLWriter &xmlFile) const
          // editing operation that caused this, not fixing
          // the problem but moving the point of detection earlier if we
          // find a reproducible case.
+         using namespace BasicUI;
          auto sMsg =
             XO("Sequence has block file exceeding maximum %s samples per block.\nTruncating to this maximum length.")
                .Format( Internat::ToString(((wxLongLong)mMaxSamples).ToDouble(), 0) );
-         AudacityMessageBox(
+         ShowMessageBox(
```

Comment:
Out of scope, yet accessing UI from here isn't a good thing

Reply:
I agree, but as my commit comment said, this is a piece of defensive programming against a very unlikely file corruption.

It's harmless now.